plugins/invisible-typing/src/Settings.tsx

```diff
@@ -16,7 +16,7 @@ export function Settings(): React.ReactElement {
       </SwitchItem>
       <SwitchItem
         {...util.useSetting(cfg, "invisible", true)}
-        note="When enabled, invisibility applies to all channels. When disabled, invisibility is channel based.">
+        note="Whether invisibility applies to all channels. (Serves as default for channel's state when invisiblity is channel based)">
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Updated note provides clearer explanation of the global invisibility setting.

This change helps users understand the dual role of the global invisibility setting, both as a universal setting and as a default for channels without a specific setting.

```suggestion
        note="This setting controls invisibility across all channels. If enabled, you'll be invisible when typing in any channel. If disabled, invisibility can be customized per channel, with this setting acting as the default behavior.">
```